src/components/map/BiodivInformation.vue

```diff
@@ -14,15 +14,16 @@
       <span>{{ months.join(", ") }}</span>
     </p>
 
-    <!-- TODO : find a light way to securely show this content -->
-    <p>{{ description }}</p>
+    <!-- content formatting done with "he" html entities decoder -->
+    <p v-for="descPar of descriptionParagraphs">{{ descPar }}</p>
```

Comment:
`v-for` needs a `key` attribute

```suggestion
    <p v-for="(descPar, i) of descriptionParagraphs" :key="i">{{ descPar }}</p>
```